std/utf.d

```diff
@@ -24,6 +24,7 @@ import std.meta;    // AliasSeq
 import std.range.primitives;
 import std.traits;  // isSomeChar, isSomeString
 import std.typecons : Flag;
+import std.exception : basicExceptionCtors;
```

Comment:
Done and pushed. While I was at it, I removed the selective import of `Flag` too. Hope that's not a problem...